cray-cmstools-crayctldeploy.spec

```diff
@@ -83,20 +83,31 @@ echo %{install_venv_python_base_dir} | tee -a INSTALLED_FILES
 cp -prv %{local_venv_python_base_dir}/* %{buildroot}%{install_venv_python_base_dir}
 find %{local_venv_python_base_dir} -print | sed 's#^%{local_venv_python_base_dir}#%{install_venv_python_base_dir}#' | tee -a INSTALLED_FILES
 
-# Add script to launch the barebones test in /opt/cray/tests/integration/csm
+# Add script to launch the cmstools test in /opt/cray/tests/integration/csm
 install -m 755 -d %{buildroot}/opt/cray/tests/integration/csm/
 echo /opt/cray/tests/integration/csm | tee -a INSTALLED_FILES
 # If the RPM contains just a single Python version, then we can use a simple symlink.
-# Otherwise we should use the run_barebones_image_test.sh script
+# Otherwise we should use the run_cmstools_test.sh script
+# barebones_image_test
 %if %{num_py_versions} == 1
 pushd %{buildroot}/opt/cray/tests/integration/csm
 ln -s ../../../../..%{install_venv_python_base_dir}/*/%{cmstools_venv_name}/bin/barebones_image_test barebones_image_test
 popd
 %else
-install -m 755 run_barebones_image_test.sh %{buildroot}/opt/cray/tests/integration/csm/barebones_image_test
+install -m 755 run_cmstools_test.sh barebones_image_test %{buildroot}/opt/cray/tests/integration/csm/barebones_image_test
 %endif
 echo /opt/cray/tests/integration/csm/barebones_image_test | tee -a INSTALLED_FILES
 
+# cfs_sessions_rc_test
+%if %{num_py_versions} == 1
+pushd %{buildroot}/opt/cray/tests/integration/csm
+ln -s ../../../../..%{install_venv_python_base_dir}/*/%{cmstools_venv_name}/bin/cfs_sessions_rc_test cfs_sessions_rc_test
+popd
+%else
+install -m 755 run_cmstools_test.sh cfs_sessions_rc_test  %{buildroot}/opt/cray/tests/integration/csm/cfs_sessions_rc_test
```

Comment:
This line also does not seem like it will work. I don't see any source file in the repo named `cfs_sessions_rc_test`, but this line is trying to install it.